<ui version="4.0" >
 <class>BackgroundDialog</class>
 <widget class="QDialog" name="BackgroundDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string> Background Settings ...</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="colorGroupBox" >
       <property name="title" >
        <string>Color</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="imageLabelPreview" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeColorButton" >
          <property name="text" >
           <string>Change color</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BackgroundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>298</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BackgroundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>366</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
